core/input/vkeys.js

```diff
@@ -58,6 +58,7 @@ export default {
     0x63: 'Numpad3',
     0x64: 'Numpad4',
     0x65: 'Numpad5',
+    0x0c: 'Numpad5', // ie11 sends evt.keyCode 12
```

Comment:
I was thinking more along the lines of when this happens. :)
E.g. "this is sent when numlock is off".